src/App.vue

```diff
@@ -967,4 +987,28 @@ function getMultilines() {
     .filter((line) => /\w/.test(line)) // removes any empty lines
     .map((line) => line.replace(/\b\n+|\n+\b/g, ""));
 }
+
+// the function of this code is to ai to help select a relevant template -SueAnn Van Valkenburg
+async function selectTemplate() {
+  const userPrompt = this.formData.prompt;
+  console.log("Prompt Text:", userPrompt);
+  try {
+    if (userPrompt != "") {
+      this.inputError = false;
+      const selectedTemplateText = await aiSelectTemplate(userPrompt);
+      console.log(`template : ${selectedTemplateText}`);
+
+      this.formData.input = selectedTemplateText;
+    } else {
+      this.inputError = true;
+    }
```

Comment:
If the userPrompt is just an empty string, do we really want to display that as an error?

Reply:
I guess the variable name is a little misleading it just changes the textbox outline to red so they can have some feedback. I also did want to send empty strings as a prompt to the api because every call to the api costs. I can show you it during the demo and if you still don't like it I can change it.